romana-install/roles/cache/apply/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# OS Files
+- include: "{{ item }}"
+  with_first_found:
+    - files:
+      - "repo_{{ ansible_distribution|lower }}_{{ ansible_distribution_release|lower }}.yml"
```

Comment:
A bit of both, but mostly the convenient skip notation.
The effect is when doing an install for a different distro (eg: centos), the task gets skipped without needing to specifically check for it.
Other uses have more of a heirarchy of first_found cases for specific distro+release, distro, then more-generic "family" of distros.